rskj-core/src/test/java/co/rsk/net/sync/PeerAndModeDecidingSyncStateTest.java

```diff
@@ -19,7 +37,7 @@
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.*;
 
-public class DecidingSyncStateTest {
+public class PeerAndModeDecidingSyncStateTest {
```

Comment:
hmm, isn't the name too descriptive ? :)

Reply:
Well, I think `DecidingSyncState` was not descriptive enough... When I started checking Peer Scoring it was not clear to me what was being "decided" there.

I think it does not harm to be a bit more descriptive and say what is being "decided".